src/Templates/src/templates/maui-blazor/.template.config/.gitignore

```diff
@@ -0,0 +1,3 @@
+# This file is generated by the build from template.in.json
+# to substitute a product version number.
+template.json
```

Comment:
Can these `template.json` files be generated in the `obj` directory instead? Then we wouldn't need to do a `.gitignore` or have to exclude `.gitignore` in the Microsoft.Maui.Templates.csproj.

Reply:
Yep! I just pushed an update for that. I was being lazy 😁